axes/decorators.py

```diff
@@ -440,7 +441,12 @@ def check_request(request, login_unsuccessful):
     if failures >= FAILURE_LIMIT and LOCK_OUT_AT_FAILURE and user_lockable:
         # We log them out in case they actually managed to enter the correct
         # password
-        if hasattr(request, 'user') and request.user.is_authenticated():
+        if django.VERSION[:2] < (2,):
+            is_authenticated = request.user.is_authenticated()
```

Comment:
we need to check if the request has the user attribute first, else the exception is raised (`AttrinuteError`), right?

Reply:
You are so right. Fixed it.